templates/details/libraries/libraries_layout.html

```diff
@@ -101,4 +118,68 @@ <h3>{{ docstrings["name"] }}</h3>
       </div>
     </div>
   </div>
+{% endblock %}
+
+{% block page_scripts %}
+  <script>
```

Comment:
Not a big deal, but I would rather put this JS in the libraries JS file https://github.com/canonical-web-and-design/charmhub.io/tree/master/static/js/src/public/details/libraries

Reply:
I agree that we should have less JS in templates. I think this is a larger discussion though, as we have this task to deal with https://github.com/canonical-web-and-design/snapcraft.io/issues/1245 which involves us moving away from the pattern we currently have for loading scripts.